docs/users/cli-usage.md

```diff
@@ -134,6 +132,11 @@ ddev config
 
 _Note: ddev config will prompt you for a project name, docroot, and project type._
 
+* the project name will be part of the URL, so make sure to avoid whitespace, underscores and special characters to avoid problems down the road.
+* after the command run through, prepare to edit the generated config.yaml file:
```

Comment:
Two change requests:
* Grammar: s/run through/runs through/
* Style: Capitalize first word of each bullet/sub-bullet.